plugins/modules/win_ntp.ps1

```diff
@@ -0,0 +1,183 @@
+#!powershell
+
+# Copyright: (c) 2020, Sakar Mehra <sakarmehra100@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#Requires -Module Ansible.ModuleUtils.Legacy
```

Comment:
We are trying to avoid using `Ansible.ModuleUtils.Legacy` for any new modules. Using the `Ansible.Basic` C# util offers numerous advantages over the old style like:

* Module invocation reporting
* More validation options, especially for list or dict types
* Enables sanity tests to check that the docs match up with the module options

Let me know if you need any help trying to convert this over to the newer style. If you need to see other modules that use this wrapper I suggest you look at [win_certificate](https://github.com/ansible-collections/community.windows/blob/main/plugins/modules/win_credential.ps1#L9-L53).